pkg/labels/labels.go

```diff
@@ -151,17 +155,37 @@ type Label struct {
 // Labels is a map of labels where the map's key is the same as the label's key.
 type Labels map[string]*Label
 
-// String returns the map of labels as human readable string
-func (l Labels) String() string {
-	str := ""
+// GetPrintableModel turns the Labels into a sorted list of strings
+// representing the labels, with CIDRs deduplicated (ie, only provide the most
+// specific CIDR).
+func (l Labels) GetPrintableModel() (res []string) {
```

Comment:
Since we have "except" in the CIDR definitions for policies, is it possible we would return too few most-specific-CIDR ranges when using this for a policy/identity print? (It should be fine if used on an endpoint, I think). I think you mention this in #3835 but I'm not sure that I fully understand the interactions here and what is left to be pulled in.

If the above can happen, and is a problem, I think we can sort it out by making `cidr` a list, and then check element in it for `ones > prefixLength` and also `cidr[i].Contains(ipnet)`. If it matches none add it to the list, if it matches one, override it (so, overwrite an entry in cidr if ipnet is more specific than that entry but add a new entry to cidr if ipnet is outside the range of all the current most specific ranges)